src/css/components/modal.scss

```diff
@@ -0,0 +1,20 @@
+
+@import "../override_vars.scss";
+@import "../core.scss";
+
+// faux modal for now, but maybe real later?
+
+.actions {
```

Comment:
I made something similar to this here: https://github.com/18F/cg-style/pull/88/files#diff-138dae25243f1dd1405b1ecacbb90189R7.

One difference in mine: I didn't want my box to expand to the full width of the page. I was thinking I could bring my stuff into this file with as something like `actions-small` or something.